packages/flutter_tools/test/version_test.dart

```diff
@@ -66,7 +66,14 @@ void main() {
     });
 
     testUsingContext('prints nothing when Flutter installation looks fresh', () async {
-      fakeData(mockProcessManager, mockCache, localCommitDate: _upToDateVersion);
+      fakeData(
+        mockProcessManager,
+        mockCache,
+        localCommitDate: _upToDateVersion,
+        // Server will be pinged because we haven't pinged within last x days
+        expectServerPing: true,
```

Comment:
This test never used to cause a server ping, but since it doesn't provide a local stamp, we do now ping the server so I had to tweak the expectations. I could also have provided a local stamp, but it felt like that was covered better in the other tests already.